### PR TITLE
Blog post: Introducing Micro Server

### PR DESCRIPTION
**Pull request by crufter**

### CODE REVIEW

File: blog/web/content/_posts/2020-05-04-introducing-micro-server.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout:	post
+title:	Introducing Micro Server
+date:	2020-05-04 17:00:00
+---
+<br>
+# Introducing Micro Server
+
+In 2016, `go-micro`, a Go microservices framework was announced. Today we introduce `micro server`, which builds on top of `go-micro`, and enables you to run and manage microservices at ease, both locally and across different environments. Wether you are running `micro server` locally with zero dependencies (using memory or files), or on k8s, databases and other third party tools, `micro server` and the micro cli should provide a straightforward and uniform experience.
```
Comment (ben-toogood) [inline blog/web/content/_posts/2020-05-04-introducing-micro-server.md:9]: I'd change "or on k8s, databases and other third party tools" to "or on k8s (with databases and other third party tools)"

File: blog/web/content/_posts/2020-05-04-introducing-micro-server.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+For those who prefer action to words, the [getting started guide](TODO LINK TO GETTING STARTED)! can give a taste of what the `micro server` is about.
```
Comment (ben-toogood) [inline blog/web/content/_posts/2020-05-04-introducing-micro-server.md:11]: Does this link need to be added?

File: blog/web/content/_posts/2020-05-04-introducing-micro-server.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Bottling 7 years of research
```
Comment (ben-toogood) [inline blog/web/content/_posts/2020-05-04-introducing-micro-server.md:13]: I'd opt for "Combining" or "Bringing together" over "Bottling"

File: blog/web/content/_posts/2020-05-04-introducing-micro-server.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Back in 2013 at a company called Hailo - a European Uber competitor - we were one of the few companies that followed Neflix's model to build the company on microservices, which enabled us to become - at the time - the most successful taxi application in Europe.
```
Comment (ben-toogood) [inline blog/web/content/_posts/2020-05-04-introducing-micro-server.md:15]: I think this sentence is too long, i'd break it up a bit.

File: blog/web/content/_posts/2020-05-04-introducing-micro-server.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+<br>
+<div style="text-align: center; width: 100%;">
+  <img src="https://micro.mu/images/runtime10.svg" />
+</div>
+
+## Combining a decade of research
+
+At Google in 2011, Asim Aslam, the creator of Micro, experienced what it was like to build systems at Google scale. Google operated a microservices architecture before most ever knew what the term meant. They did this because of an organisational and technical need to build systems that could be developed and managed independently.
+
+In 2013 at a company called Hailo - a European Uber competitor - we were one of the few companies that followed Neflix's model to build an organisational architecture based on microservices. This enabled us to become - at the time - to be incredibly productive and arguably the most successful taxi app in Europe.
```
Comment (crufter) [inline blog/web/content/_posts/2020-05-04-introducing-micro-server.md:21]: The 'we' 'us' at Hailo might strike weird as there is no context for the reader to know many of us worked at Hailo and how Hailo vs Micro relates.

File: blog/web/content/_posts/2020-05-04-introducing-micro-server.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+Years later, ex-Hailo members brought the fruits of years of microservices R&D done at Hailo to companies like Monzo, Sixt and many others. We at the Micro team are working on bringing these ideas to the Open Source community. The source of our passion is both the fact that microservices enable companies to be successful and also that we have collectively seen many dozens of companies building similar systems from scratch. Some successfully, some not so successfully.
+
+## Complexity in the age of cloud computing
+
+We primarily exist to ensure company and developer success - the advantages of microservices became clear to the industry in the past years, but the tooling around it was and is still in infancy. Breaking up the monolith to different processes comes with an increase in [accidental complexity](https://en.wikipedia.org/wiki/No_Silver_Bullet), which, without appropriate tooling can cause non-negligible amounts of pain. Micro's mission is to make this pain go away, and unlock a whole new ear of computing where services play as nicely together as functions in a programming language do.
```
Comment (crufter) [inline blog/web/content/_posts/2020-05-04-introducing-micro-server.md:27]: era not ear

File: blog/web/content/_posts/2020-05-04-introducing-micro-server.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+We also aim to help working with different environments (ie local/custom envs/prod) and tools (different databases, orchestration systems etc.), so similar concepts and interfaces can be reused across them.
+
+There is no question that most local and production setups vastly differ.
+The differences dictated by resiliency and scaling requirements and available computing resources creates a disconnect between the different steps during the lifecycle of a code change - from the moment of its birth locally to different environments and ending up in production.
+
+A simple file backed persistent on a simple node is not enough, but similarly you might not want to install and maintain kubernetes or different databases locally.
```
Comment (crufter) [inline blog/web/content/_posts/2020-05-04-introducing-micro-server.md:34]: "not enough" in production settings

**Pull request merged**